examples/demo/site/modules/auth/templates/block_account_info.tpl

```diff
@@ -0,0 +1,60 @@
+ 	<div class="primary-tabs">
+		{if isset="$user"}
+		<h3>Account Information</h3>
+		<div>
+			<div>
+				<div>	
+			   		<label>Username:</label>  {$user.name/}
+			   	</div>
+			   	<div>	
+			   		<label>Email:</label>  {$user.email/}
+			   	</div>
+			   	<div>	
+			   		<label>Creation Date:</label>  {$user.creation_date/}
+			   	</div>
+			   	<div>	
+			   		<label>Last login:</label>  {$user.last_login_date/}
+			   	</div>	
+			   	<div>
+			   		<form method="get" action="{$site_url/}account/{$strategy/}">
+    					<button type="submit">Logout</button>
+					</form>
+			   	</div>	
+			</div>
+    	</div>
+    	<hr>
+    	<h4>Roles</h4>
+    	<div>
+    		{foreach item="ic" from="$roles"}
```

Comment:
"ic" is ok for Eiffel, but here ... this is not the cursor, it is the role item itself.
so better name it  "role"